holidays/countries/united_states.py

```diff
@@ -49,11 +49,34 @@ class UnitedStates(ObservedHolidayBase, ChristianHolidays, InternationalHolidays
     Frances Xavier Cabrini Day:
         - https://leg.colorado.gov/sites/default/files/2020a_1031_signed.pdf
 
-    Washington's Birthday (GA):
-        - https://www.gasupreme.us/court-information/holidays-2/
-
-    Washington's Birthday (IN):
-        - https://www.in.gov/spd/benefits/state-holidays/
```

Comment:
@parkerbxyz, thank you for such a wide list of official sources! It would also be useful to keep these two links, especially for GA (which is not listed in your sources).